### PR TITLE
Analysis notebooks

### PR DESCRIPTION
**Pull request by ben-heil**
This PR contains notebooks of analyses I ran to better understand the relationship of pageranks between fields

Changes:
- Added the `heading_similarity` notebook to see how many papers overlap between fields and how that affects the pagerank 
- Added journal comparison plots to the `interfield_differences` notebook
- Wrote the `pagerank_inflation` notebook to figure out why the papers with low pageranks have high percentile scores

### CODE REVIEW

File: notebooks/heading_similarity.ipynb [hunk 1]
```diff
@@ -0,0 +1,1668 @@
+{
```
Comment (jjc2718) [inline notebooks/heading_similarity.ipynb:1]: <p>How is this different from the code above? It looks like the plot is more or less the same (at least as far as I can tell).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (jjc2718) [inline notebooks/heading_similarity.ipynb:1]: <p>Are all of the counts strictly smaller? (Looks that way to me but I haven't checked in detail)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (jjc2718) [inline notebooks/heading_similarity.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    plot += ylab('paper overlap fraction')
</pre><p>Should this be 'number of overlapping papers'?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

**Review by jjc2718 — APPROVED** [review]
Looks good! Just a few small comments (through ReviewNB).

File: notebooks/heading_similarity.ipynb (hunk 1 above)
Comment (ben-heil) [reply notebooks/heading_similarity.ipynb:1]: <p>Weird, it looks like the cells got duplicated. I'll go remove the extras</p>

Comment (ben-heil) [reply notebooks/heading_similarity.ipynb:1]: <p>Yep, they're strictly smaller</p>

Comment (ben-heil) [reply notebooks/heading_similarity.ipynb:1]: <p>Yep, forgot to update the label</p>

**Comment by ben-heil** [comment]
<p>So everything below the line on the median plot will have a &gt;50 percentile. Since there is a lot of variance where the shuffled pagerank is low and the minimum is bounded, there is a lot more room for things to have high percentiles. That said, it looks like there might be something going on around the pagerank ~= 1e-13 mark, so I'll keep digging</p>

---

 <a href='https://app.reviewnb.com/greenelab/indices/pull/8/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request merged**